index.html

```diff
@@ -3842,6 +3846,107 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
 </aside>
 </section>
 
+<section class="informative changed"><h2>JSON Literals</h2>
+  <p>At times, it is useful to include JSON within JSON-LD that is not interpreted as JSON-LD.
+    Generally, a JSON-LD processor will ignore properties which don't map to <a>IRIs</a>,
+    but this causes them to be excluded when performing various algorithmic transformations.
+    But, when the data that is being described is, itself, JSON, it's important that
+    it survive algorithmic transformations.</p>
+
+  <p>By defining a term with <code>@type</code> <code>@json</code>,
```

Comment:
Phrased like that, it sounds like the processor is the one defining the term with `@type` `@json`. I suggest rephrasing it as follows:
```suggestion
  <p>When a term is defined with <code>@type</code> <code>@json</code>,
```